src/styles/index.scss

```
@@ -1,3 +1,10 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
+@import './blocks/variables';
+@import './blocks/card';
+@import './blocks/button';
+@import './blocks/stars';
```

```suggestion
@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
@import './blocks/variables';
@import './blocks/card';
@import './blocks/button';
@import './blocks/stars';

body {
  margin: 0;
  font-family: Roboto, sans-serif;
}
```

Let`s firstly import files and then write your styles